azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecServerModule.java

```diff
@@ -30,11 +37,49 @@
  */
 public class AzkabanExecServerModule extends AbstractModule {
 
+  private static final Logger logger = Logger.getLogger(AzkabanExecServerModule.class);
+  private static final String EVENT_REPORTER_CLASS_PARAM = "event.reporter.class";
+  private static final String EVENT_REPORTER_ENABLED = "event.reporter.enabled";
+
   @Override
   protected void configure() {
     bind(ExecutorLoader.class).to(JdbcExecutorLoader.class);
     bind(AzkabanExecutorServer.class).in(Scopes.SINGLETON);
     bind(TriggerManager.class).in(Scopes.SINGLETON);
     bind(FlowRunnerManager.class).in(Scopes.SINGLETON);
   }
+
+  @Inject
+  @Provides
```

Comment:
Is the instance designed to be a singleton? If so annotate with @Singleton?
https://github.com/google/guice/wiki/Scopes

Reply:
For the current usage, marking it with `@singleton` makes sense.

Reply:
done.